devicetypes/smartthings/zwave-button.src/zwave-button.groovy

```diff
@@ -0,0 +1,148 @@
+/**
+ *  Z-Wave Button
+ *
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition (name: "Z-Wave Button", namespace: "smartthings", author: "SmartThings") {
+		capability "Button"
+		capability "Battery"
+		capability "Sensor"
+		capability "Health Check"
+		capability "Configuration"
+
+		fingerprint mfr: "010F", prod: "0F01", model: "1000", deviceJoinName: "Fibaro Button"
+		fingerprint mfr: "0086", prod: "0001", model: "0026", deviceJoinName: "Aeotec Panic Button"
```

Comment:
We already have this fingerprint in https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/aeon-key-fob.src/aeon-key-fob.groovy

Reply:
so this PR should also include removing that fingerprint from that DTH if that's the intention.

Reply:
Yup, that was intended. That is, if I understood Eric correctly in WWST-2021.